src/benchmarks/real-world/Microsoft.ML.Benchmarks/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Exporters.Json;
+using BenchmarkDotNet.Horology;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using MicroBenchmarks;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.ML.Benchmarks
+{
+    class Program
+    {
+        /// <summary>
+        /// execute dotnet run -c Release and choose the benchmarks you want to run
+        /// </summary>
+        /// <param name="args"></param>
+        static int Main(string[] args)
+            => BenchmarkSwitcher
+                .FromAssembly(typeof(Program).Assembly)
+                .Run(args, GetConfig())
+                .ToExitCode();
+
+        private static IConfig GetConfig()
+            => DefaultConfig.Instance
+                .With(Job.Default
+                    .WithWarmupCount(1) // 1 warmup is enough for our purpose
+                    .WithIterationTime(TimeInterval.FromMilliseconds(250)) // the default is 0.5s per iteration, which is slighlty too much for us
+                    .WithMinIterationCount(15)
+                    .WithMaxIterationCount(20) // we don't want to run more that 20 iterations
+                    .AsDefault()) // tell BDN that this are our default settings
+                .WithArtifactsPath(Path.Combine(
+                    Path.GetDirectoryName(typeof(Program).Assembly.Location), "BenchmarkDotNet.Artifacts"))
+                .With(MemoryDiagnoser.Default) // MemoryDiagnoser is enabled by default
+                .With(new OperatingSystemFilter())
+                .With(JsonExporter.Full) // make sure we export to Json (for BenchView integration purpose)
+                .With(StatisticColumn.Median, StatisticColumn.Min, StatisticColumn.Max)
+                .With(TooManyTestCasesValidator.FailOnError)
+                .With(MandatoryCategoryValidator.FailOnError);
```

Comment:
same as above: we don't need categories if there is only one category in given project

Reply:
See above.